typescript/static-site/static-site.ts

```diff
@@ -4,6 +4,7 @@ import route53 = require('@aws-cdk/aws-route53');
 import s3 = require('@aws-cdk/aws-s3');
 import ssm = require('@aws-cdk/aws-ssm');
 import cdk = require('@aws-cdk/cdk');
+import targets = require('@aws-cdk/aws-route53-targets/lib');
```

Comment:
This should be just `import targets = require('@aws-cdk/aws-route53-targets');`, similar to the imports above.